microsoft-edge/webview2/reference/javascript/sharedbufferreceivedevent.yml

```diff
@@ -0,0 +1,131 @@
+### YamlMime:TSType
+name: SharedBufferReceivedEvent
+uid: 'WebView2Script!SharedBufferReceivedEvent:class'
+package: WebView2Script!
+fullName: SharedBufferReceivedEvent
+summary: >-
+  Event object for the `chrome.webview.sharedbufferreceived` event. This event is dispatched when
+  `CoreWebView2.PostSharedBufferToScript` is successfully called.
+remarks: |-
+
+
+  #### Examples
+
+  For Example, if we want to send data to script for one time read only consumption,
+
+  In native host app code, set data into the shared memory.
+
+  ```cpp
+  wil::com_ptr<ICoreWebView2ExperimentalEnvironment10> environment;
+  CHECK_FAILURE(
+      m_appWindow->GetWebViewEnvironment()->QueryInterface(IID_PPV_ARGS(&environment)));
+
+  wil::com_ptr<ICoreWebView2ExperimentalSharedBuffer> sharedBuffer;
+  CHECK_FAILURE(environment->CreateSharedBuffer(bufferSize, &sharedBuffer));
+  // Set data into the shared memory via IStream.

```

Comment:
```suggestion
  // Add data to shared memory using IStream.
```

Reply:
Not feeling strong but "by using" does not exactly seem like a synonym for "via" here. Is it okay if we keep "via" here?

Reply:
```suggestion
  // Add data to shared memory via IStream.
```